src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static List<User> usersForCheck;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void classImplementation() {
+        usersForCheck = new ArrayList<>();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void clearingListOfUsers() {
+        usersForCheck.clear();
+        Storage.people.clear();
+    }
+
+    @BeforeEach
+    void fillingListWith_5_ValidUsers() {
```

Comment:
It is better to create one user and change the data in it to the ones you need where you are testing